sql/analyzer/optimization_rules.go

```diff
@@ -1,7 +1,7 @@
 package analyzer
 
 import (
-	errors "gopkg.in/src-d/go-errors.v1"
+	"gopkg.in/src-d/go-errors.v1"
```

Comment:
Actually changes in this file you can discard.

Reply:
It was the result of gofmt over those files which doesn't hurt, but I'll discard to keep the PR tidier.

Reply:
This is a change goimports usually does. Instead of having to manually revert it every time I'd just keep it.